build/SolutionWideAnalyzerConfig.ruleset

```diff
@@ -160,4 +157,11 @@
     <Rule Id="CA3076" Action="None" />
     <Rule Id="CA3077" Action="None" />
   </Rules>
+  <Rules AnalyzerId="StyleCop.Analyzers" RuleNamespace="StyleCop.Analyzers">
+    <Rule Id="SA1101" Action="None" />
+    <Rule Id="SX1101" Action="Warning" />
```

Comment:
I propose to set this to Error, because we currently have quite a few warnings (sadly the Xamarin stuff generates some which they didn't fix in ages). So if you set it to warning, it will be ignored. 

Ideally we will be able to have no more warnings and enable "treat warnings as errors" or smth, but until then - error!

Reply:
I'm trying to do a bunch of PRs this week to get our warning count for little stuff and get the nitpicky hygiene stuff cleared out.  Will look at moving 1101 to error.